examples/restify/src/integration-test/groovy/restify/BulkRestApiSpec.groovy

```diff
@@ -1,22 +1,21 @@
 package restify
 
+import gorm.tools.job.JobState
 import gorm.tools.rest.JsonParserTrait
 import gorm.tools.rest.client.OkHttpRestTrait
 import grails.gorm.transactions.Rollback
 import grails.testing.mixin.integration.Integration
 import okhttp3.Response
 import org.springframework.http.HttpStatus
-import spock.lang.Ignore
 import spock.lang.Specification
+import yakworks.rally.job.Job
 import yakworks.rally.orgs.model.Org
 
 @Rollback
 @Integration
-
 class BulkRestApiSpec extends Specification implements OkHttpRestTrait, JsonParserTrait {
     String path = "/api/rally/org/bulk?jobSource=Oracle"
 
-    //XXX https://github.com/yakworks/gorm-tools/issues/348 can we test Job here?
     void "verify bulk create and sanity check response"() {
```

Comment:
test jobSource id and that the job.source got populated with the url that called it